frontend/src/styles/buttons/ScrollToTopButton.module.css

```diff
@@ -0,0 +1,23 @@
+.ScrollToTopButton {
+    position: fixed;
+    display: inline-block;

```

Comment:
why this line?

Reply:
@heylookitsme If I remember right, the inline-block CSS was something I copied from https://www.tutorialrepublic.com/faq/how-to-change-image-on-hover-with-css.php